src/rpcnet.cpp

```diff
@@ -365,14 +365,20 @@ static Array GetNetworksInfo()
         enum Network network = static_cast<enum Network>(n);
         if(network == NET_UNROUTABLE)
             continue;
+
+        proxyType proxyDefault;
+        GetProxy(network, proxyDefault);
         proxyType proxy;
-        Object obj;
         GetProxy(network, proxy);
+
+        Object obj;
         obj.push_back(Pair("name", GetNetworkName(network)));
         obj.push_back(Pair("limited", IsLimited(network)));
         obj.push_back(Pair("reachable", IsReachable(network)));
         obj.push_back(Pair("proxy", proxy.IsValid() ? proxy.proxy.ToStringIPPort() : string()));
         obj.push_back(Pair("proxy_randomize_credentials", proxy.randomize_credentials));
+        if (network != NET_IPV4)
+            obj.push_back(Pair("proxy-from-default", proxy.proxy == proxyDefault.proxy));
```

Comment:
Let's remove `proxy-from-default`, it's not useful information and will always be true anyway according to the above code.